plugins/disk/quota2percent_

```diff
@@ -0,0 +1,249 @@
+  Id=${0##*_}
+
+# Check if device exist
+  if ! df "/dev/$Id" &> /dev/null; then
+     echo "The device /dev/$Id does not exist!" >&2
+     exit 128
+  fi
+
+###################################################
+# Data reading sections                           #
+###################################################
+
+# Reading the quotes for the selected device, using repquota
+  if repquota "/dev/$Id" &> /dev/null; then     
+     readarray Quotas < <( repquota "/dev/$Id" | grep " -- " )
+  else
+     echo "No limitations administered via 'quota' for $Id" >&2
+     # If no limitatitons administered: create a dummy
+     Quotas[0]="root -- 1 1 1 1 1 1"
+  fi
+
+  readarray Totals < <( df "/dev/$Id" )
+
+# Get the count of Users 
+   Users=${#Quotas[@]}
+
+
+###################################################
+# Munin Configuration Section                     #
+###################################################
+
+  if [ "$1" = "config" ]; then
+
+     # Localisation of the graphic texts 
+     case $Language in
+          de)
+            echo graph_title  "Quota-Hard-Limit von $Id"
+            echo graph_vlabel "Nutzung in % Hardlimit"
+            echo graph_info   "Die Grafik zeigt die Belegung des durch Quota reglementierten Speicherplatzes für alle regulären Nutzer (UID >= $Min_UID) in Prozent des Hardlimits."
+            Total_txt="Su. aller Nutzer"
+            Total_info="Inklusive Systemnutzer (UID < $Min_UID)"
+            ;;
+          es) 
+            echo graph_title  "Cuota de límite absoluto de $Id" 
+            echo graph_vlabel "el % de uso del límite duro" 
+            echo graph_info   "El gráfico muestra la disponibilidad de espacio regulado por cuotas para todos los usuarios regulares (UID >= $Min_UID) como porcentaje de límites duros."
+            Total_txt="Suma de todos los usuarios "
+            Total_info="La inclusión de usuario del sistema (UID < $Min_UID) "
+            ;;
+           *)
+            echo graph_title  "quota hard limit of $Id"
+            echo graph_vlabel "Usage in %"
+            echo graph_info   "The graphic shows the allocation of the quota-regulated storage space for all regular users (UID >= $Min_UID) as a percentage of the hard limit ."
+            Total_txt="all users"
+            Total_info="system users (UID < $Min_UID) included" 
+            ;;
+     esac
+
+     # Defaults configuration 
+       echo    graph_category disk
+       echo    graph_args --lower-limit 0 --upper-limit 100
+       echo    graph_printf %5.2lf %%
+       echo    graph_scale  no
+
+     # For each quota user fetch his real name, solve the root problem,  Output the configuration data 
+       for((i=0; i<"$Users"; i++));do
+         Quota=( ${Quotas[$i]} )
+         UserName=${Quota[0]}
+         Passwd=$(grep "$UserName" /etc/passwd)
+         IFS=':' Infos=($Passwd) 
+         UserInfo=${Infos[4]}
+         UserInfo=${UserInfo%%,*}
+
+         UserName="$(clean_fieldname "$UserName")"
```

Comment:
Since the result of this assignment is not really the `UserName` anymore, I would suggest to name it `fieldname` instead. This would avoid confusion.